docs/4.3/enterprise/quickstart-enterprise.md

```diff
@@ -315,6 +320,104 @@ Any SAML-compliant provider can be configured with Teleport by following the
+
+# start teleport quickstart using docker-compose
+docker-compose -f teleport-ent-quickstart.yml up
+```
+
+- The `docker-compose` quickstart will automatically create a config file for you at `./docker/teleport/config/teleport.yaml`
+- This config is mounted into the container under `/etc/teleport/teleport.yaml`
+- It will also start `teleport` using this config file, with Teleport's data directory set to `./docker/teleport/data` and mounted under `/var/lib/teleport`
+- It will mount your license file (named `license.pem`) from the current directory into the Docker container
+- By default, `docker-compose` will output Teleport's logs to the console for you to observe.
+    - If you would rather run the Teleport container in the background, use `docker-compose -f teleport-ent-quickstart.yml up -d`
+    - You can stop the Teleport container using `docker-compose -f teleport-ent-quickstart.yml down`
+
+### Quickstart using docker run
+
+If you'd prefer to complete these steps manually, here's some sample `docker run` commands:
+
+```bash
+# create local config and data directories for teleport, which will be mounted into the container
+mkdir -p ~/teleport/config ~/teleport/data
+
+# download your license file from the Gravitational dashboard and put it in the correct directory
+# the file needs to be named license.pem
+cp ~/downloads/downloaded-license.pem ~/teleport/data/license.pem
+
+# generate a sample teleport config and write it to the local config directory
+# this container will write the config and immediately exit - this is expected
+docker run --hostname localhost --rm \
+  --entrypoint=/bin/sh \
+  -v ~/teleport/config:/etc/teleport \
+  {{teleport.latest_ent_docker_image}} -c "teleport configure > /etc/teleport/teleport.yaml"
+
+# change the path to the license file in the sample config
+sed -i 's_/path/to/license-if-using-teleport-enterprise.pem_/var/lib/teleport/license.pem_g' ~/teleport/config/teleport.yaml
+
+# start teleport with mounted license, config and data directories, plus all ports
+docker run --hostname localhost --name teleport \
+  -v ~/teleport/config:/etc/teleport \
+  -v ~/teleport/data:/var/lib/teleport \
+  -p 3023:3023 -p 3025:3025 -p 3080:3080 \
+  {{teleport.latest_ent_docker_image}}
+```
+
+### Creating a Teleport user when using Docker quickstart
+
+To create a user inside your Teleport Enterprise container, use `docker exec`.
+
+This example command will create a Teleport user called `testuser` which has the `admin` role. Feel free to change these to suit your needs.
+
+```bash
+docker exec teleport tctl users add testuser --roles=admin
+```
+
+When you run this command, Teleport will output a URL which you must open to complete the user signup process:
+
+```bash
+User testuser has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h0m0s:
+https://localhost:3080/web/invite/4f2718a52ce107568b191f222ba069f7
```

Comment:
Does the user require any extra steps to setup TLS for Docker on Local host?

Reply:
I don't think so.
Docker container just exposes a TCP port. What runs on top of that should be invisible to docker.

Reply:
Yeah, it's likely that the user will get an insecure warning in their browser that they'll have to click through but that's the case with regular Teleport setups too.

Reply:
How about we recommend / add a note about using https://github.com/FiloSottile/mkcert ?